### ISSUE TITLE
Deprecated modules/API

### ISSUE THREAD

**Issue by geoced** [get-itips/msshells#608]
Hello,

I stumbed upon your site and found it useful, but I have a few questions:

Aren't AzureAD, AzureADPreview and MSOnline deprecated and being replaced with Microsoft.Graph ?

Also, I don't see the AzureRm module in here, but I think it has also been replaced with Az ?

Thanks !

**Comment by get-itips** [comment]
Hello!
Thanks for finding the site useful and leaving your feedback, I appreciate that.

Yes, those modules are either close to being deprecating or already deprecated, but still a lot referenced in documentation and official procedures, so decided to keep them.

I will review if I should include AzureRm

Thanks again!

**Comment by geoced** [comment]
Thanks !

Maybe adding a footnote on deprecated modules ? Or adding a Status column (Maintained/Deprecated) ?

MS says "_We recognize that the legacy PowerShell modules are required for some scenarios not yet available in Microsoft Graph PowerShell SDK. Therefore, we are postponing the deprecation date for MS Online, AzureAD, and AzureAD Preview PowerShell modules to March 30, 2024._"

But I think the latest update of AzureADPreview broke Connect-AzureAD, I had to downgrade to keep using that module.

See here: https://techcommunity.microsoft.com/t5/microsoft-entra-azure-ad-blog/important-azure-ad-graph-retirement-and-powershell-module/ba-p/3848270

**Comment by get-itips** [comment]
Not a bad idea at all!

@robdy what do you think?

**Comment by robdy** [comment]
Yeah, I agree that we can mention module status somewhere. Footnote makes more sense than a column as we already have plenty of columns. For me, ideally - footnote that also appears on hover ;)

**Comment by get-itips** [comment]
Yes, you are right, too many columns already, I'll work in a footnote
